### PR TITLE
feat: add gdal_translate options (TDE-80) (TDE-260)

### PR DESCRIPTION
**Pull request by amfage**
Sets docker container environment variable GDAL_DISABLE_READDIR_ON_OPEN=EMPTY_DIR for GDAL COG creation.
[https://linzgovt.atlassian.net/browse/TDE-260](https://linzgovt.atlassian.net/browse/TDE-260)

Sets the following options:
OVERVIEW_COMPRESS=JPEG
OVERVIEW_QUALITY=90
[https://linzgovt.atlassian.net/browse/TDE-80](https://linzgovt.atlassian.net/browse/TDE-80)

### CODE REVIEW

**Review by dwsilk — APPROVED** [review]

File: topo_processor/cog/create_cog.py [hunk 1]
```diff
@@ -5,27 +5,27 @@
 from topo_processor.util.aws_credentials import Credentials, get_credentials
 
 
-def create_cog(input_path: str, output_path: str, compression_method: str) -> str:
+def create_cog(input_path: str, output_path: str, compression_method: str, overview_compression_method: str) -> str:
```
Comment (paulfouquet) [inline topo_processor/cog/create_cog.py:8]: Is there a reason to have an argument for this specific `gdal` parameter? i.e. are we anticipating something?

**Review by paulfouquet — COMMENTED** [review]
Just a quick question to avoid adding complexity if it's not necessary.

File: topo_processor/cog/create_cog.py (hunk 1 above)
Comment (paulfouquet) [reply topo_processor/cog/create_cog.py:8]: IMO we generally should make this kind of change when it actually happens.

Comment (amfage) [reply topo_processor/cog/create_cog.py:8]: Discussion with @dwsilk @paulfouquet - will hard code both `compression` and `overview_compression`.

**Review by paulfouquet — APPROVED** [review]
Looks good!

**Pull request merged**